README.md

```diff
@@ -84,16 +95,63 @@ You should note that the [Push](src/main/java/com/liferay/mobile/push/Push.java)
 
 ### Receiving push notifications
 
-Once your device is registered, your app must be able to listen for notifications. 
+Once your device is registered, you have to configure both the server and the client to be able to receive push messages.
+
```

Comment:
This step has to be before registering the devices, you need the API_KEY to register them, right?